File: app/src/main/java/com/BlizzardArmory/diablo/Character/Rune.java
package com.BlizzardArmory.diablo.Character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.apache.commons.lang.builder.ToStringBuilder;

public class Rune {

    @SerializedName("slug")
    @Expose
    private String slug;
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("level")
    @Expose
    private Integer level;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("descriptionHtml")
    @Expose
    private String descriptionHtml;

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getLevel() {
        return level;
    }

    public void setLevel(Integer level) {
        this.level = level;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDescriptionHtml() {
        return descriptionHtml;
    }

    public void setDescriptionHtml(String descriptionHtml) {
        this.descriptionHtml = descriptionHtml;
    }

    @Override
    public String toString() {
        return new ToStringBuilder(this).append("slug", slug).append("type", type).append("name", name).append("level", level).append("description", description).append("descriptionHtml", descriptionHtml).toString();
    }

}

File: app/src/main/java/com/BlizzardArmory/model/diablo/character/skills/Rune.kt
package com.BlizzardArmory.model.diablo.character.skills

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName


/**
 * The type Rune.
 */
@Keep
data class Rune(

    @SerializedName("slug")
    var slug: String,

    @SerializedName("type")
    var type: String,

    @SerializedName("name")
    var name: String,

    @SerializedName("level")
    var level: Int,

    @SerializedName("description")
    var description: String,

    @SerializedName("descriptionHtml")
    var descriptionHtml: String

)